### PR TITLE
FE-200: Display SmartGraph info in Graph Viewer

### PR DESCRIPTION
**Pull request by ajurdak**
### Scope & Purpose

As SmartGraphs are not supported in the new Graph Viewer yet we display a message with a button to open the old Graph Viewer instead.

- [x] :pizza: New feature

### Checklist

- [x] Manually tested
- [x] :book: CHANGELOG entry made

#### Related Information

- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/FE-221

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/SmartGraphEmptyState.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Alert, Button, Center, Text, VStack } from "@chakra-ui/react";
+import React from "react";
+import { useGraph } from "./GraphContext";
+
+export const SmartGraphEmptyState = () => {
+    const { graphName } = useGraph();
+
+    return (
+        <Center h='100%'>
+            <Alert width="auto" background="gray.300">
```
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/SmartGraphEmptyState.tsx:10]: we can remove the background color here, also can just use a regular box, Alert not needed

File: js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/SmartGraphEmptyState.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                <VStack spacing={4} direction="row" align="center">
```
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/SmartGraphEmptyState.tsx:11]: Here we are making a Vstack into an Hstack, let's just use HStack

```suggestion
                <HStack spacing={4} align="center">
```

File: js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/GraphNetwork.tsx [hunk 3]
```diff
@@ -109,16 +111,18 @@ export const GraphNetwork = () => {
...
-          containerRef
-        }}
-      />
-      <Box id="graphNetworkWrap" height="full" backgroundColor="white">
-        {progressValue < 100 ? <Progress value={progressValue} /> : null}
-        <Box ref={visJsRef} height="calc(100% - 40px)" width="full" />
-        <GraphInfo />
-      </Box>
+      {!isSmart? <>
+        <GraphRightClickMenu
```
Comment (palashkaria) [inline js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/GraphNetwork.tsx:115]: let's avoid this nesting. we can move this into it's own `<GraphNetworkInner />` component and then do `{!isSmart ? <GraphNetworkInner /> : <SmartGraphEmptyState /> }` like in other places

**Review by palashkaria — CHANGES_REQUESTED** [review]

File: CHANGELOG [hunk 4]
```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* FE-221: Display info if a SmartGraph is opened in new Graph Viewer.
```
Comment (palashkaria) [inline CHANGELOG:4]: we probably don't need this in the changelog as this PR is raised to another branch with an open PR

**Review by palashkaria — CHANGES_REQUESTED** [review]

File: js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/GraphNetwork.tsx (hunk 3 above)
Comment (ajurdak) [reply js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/GraphNetwork.tsx]: Refactored it into a separate component.

File: js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/SmartGraphEmptyState.tsx (hunk 1 above)
Comment (ajurdak) [reply js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/SmartGraphEmptyState.tsx]: Done.

File: js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/SmartGraphEmptyState.tsx (hunk 2 above)
Comment (ajurdak) [reply js/apps/system/_admin/aardvark/APP/react/src/views/visgraphs/SmartGraphEmptyState.tsx]: Changed to Stack.

**Review by palashkaria — APPROVED** [review]

**Pull request merged**